CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 
 * Fixed the unmatched language issue of some localized strings in the feedback view. ([#4007](https://github.com/mapbox/mapbox-navigation-ios/pull/4007))
 * Added `NavigationSettings.utilizeSensorData` toggle to allow navigator to use additional device sensors data for better positioning. ([#3973](https://github.com/mapbox/mapbox-navigation-ios/pull/3973))
+* Fixed the issue of restricted layer over the whole route after the removal of continuous alternative route with a small portion of restricted road class at start. ([#4009](https://github.com/mapbox/mapbox-navigation-ios/pull/4009))
```

Comment:
To catch such issues in future we can consider adding new snapshot tests to `MapboxNavigationTests`.

Reply:
Thanks for the suggestion, I'll open a ticket to track this issue.